src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/commands.py

```diff
@@ -112,6 +115,22 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         operations_tmpl='azure.cli.command_modules.acr.check_health#{}'
     )
 
+    acr_scope_map_util = CliCommandType(
```

Comment:
Can you check if you need a table transformer for scope-maps and tokens?
Run `az acr scope-map list -r myRegistry -o table` and check if output is as expected?

Reply:
I don't think there's a need for it. Table output is looking nice